sonic-xcvrd/scripts/xcvrd

```diff
@@ -171,6 +171,14 @@ def _wrapper_get_transceiver_change_event(timeout):
             pass
     return platform_sfputil.get_transceiver_change_event(timeout)
 
+def _wrapper_get_sfp_type(physical_port):
+    if platform_chassis is not None:
```

Comment:
Suggest simplifying to `if platform_chassis:`

Reply:
Done.